libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/TurnIconHelper.kt

```diff
@@ -51,34 +48,28 @@ internal class TurnIconHelper(
     // TurnIconPair<Type, Modifier>
     private val turnIconMap: Map<TurnIconPair, Int> = mapOf(
         // When type == null and modifier == null
-        TurnIconPair(null, null) to turnIconResources.turnIconInvalid,
-        TurnIconPair("", "") to turnIconResources.turnIconInvalid,
+        TurnIconPair(null, null) to turnIconResources.turnIconTurnStraight,
+        TurnIconPair("", "") to turnIconResources.turnIconTurnStraight,
```

Comment:
This is how the response with empty type should be interpreted now, as a straight? How is this different than `TurnIconPair(null, STRAIGHT)`?

Reply:
This use case is not defined either from product or design team. It's just my interpretation to make sure in case both type and modifier are empty, then we do return something. Earlier I was returning an invalid turn icons which is same as turn straight icon. In this PR since all invalid icons have been removed, I am using turn straight.
cc @LukaszPaczos

Reply:
Gotcha, could you add an inline comment for this?